python/llm/portable-installer/README.md

```diff
@@ -0,0 +1,33 @@
+# BigDL-LLM One-Click Installer For Windows: User Guide
```

Comment:
Since we actually didn't require user to install anything, shall we call it "BigDL-LLM Portable"?

Reply:
Already done as you said. :)